src/bootstrap/src/core/build_steps/clippy.rs

```diff
@@ -195,7 +217,10 @@ impl Step for Rustc {
 
     fn make_run(run: RunConfig<'_>) {
         let crates = run.make_run_crates(Alias::Compiler);
-        let config = LintConfig::new(run.builder);
+        let config = LintConfig::new(
+            run.builder,
+            Some("src/etc/clippy_configs/compiler/clippy.toml".into()),
```

Comment:
This seems like a bit of an odd place to put clippy configs.. maybe we can move them to e.g. src/bootstrap, similar to profile files?

I'm also wondering if there's any clippy-native feature for these configs that we could be using instead? For example package-specific configs or something like that?

Reply:
Yes, location not really important, i can move it.

Lints can be set via https://doc.rust-lang.org/nightly/cargo/reference/manifest.html#the-lints-section, but not config for them https://doc.rust-lang.org/clippy/lint_configuration.html, as i understand.

Reply:
Hm, I thought we had user-configurable config for e.g. the cfg lints. Maybe those are special cased though.